src/Support/HigherOrderMessageCollection.php

```diff
@@ -30,7 +30,7 @@ public function add(string $filename, int $line, string $methodName, array $argu
     public function chain(object $target): void
     {
         foreach ($this->messages as $message) {
-            $target = $message->call($target);
+            $target = $message->call($target) ?? $target;
```

Comment:
Let's remove this for now.

Reply:
If I remove this the test won't pass anymore because `macro()` returns `null`.
It's a scenario only for the test - but I don't know how I should adjust the test to work without this? 🤔

Reply:
https://github.com/pestphp/pest/pull/109/commits/e135979f34bd6d9dcbffe5c10c4dbf4f9a2a7806